src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+/*  CARD BLOCK:  */
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
+  --dark-color: #060b35;
+  --grey-color: #616070;
```

Comment:
Better to set css variables to `:root` to make it possible to use it not only in `card__info` but in any element on the page